usersettings.py

```diff
@@ -2,9 +2,15 @@
 """
 Provide interface for persistent portable editable user settings
 """
-import os
-import ConfigParser
 import ast
+import os
+import sys
+
+PY3 = sys.version_info[0] == 3
+if PY3:
+    import configparser
+else:
+    import ConfigParser as configparser
```

Comment:
This approach will break when python4 is released. It should be re-worked like so:

    PY2 = sys.version_info[0] == 2
    if PY2:
        import ConfigParser as configparser
    else:
        import configparser